extension/css/webmail.css

```diff
@@ -1,7 +1,7 @@
 /* © 2016-2018 FlowCrypt Limited. Limitations apply. Contact human@flowcrypt.com */
 
 /* Secure Compose button */
-#flowcrypt_secure_compose_button {
+#flowcrypt_secure_compose_button_content {
```

Comment:
Let's call it `#flowcrypt_secure_compose_button_icon`, as it's really just icon, not full button content

Reply:
Sounds good

Reply:
named `content` because of this line but `flowcrypt_secure_compose_button_icon` seems good
https://github.com/FlowCrypt/flowcrypt-browser/blob/f0e097742f1b35f975cb08d46bc87f640edf6370/extension/js/common/xss-safe-factory.ts#L253